src/main/resources/db/migration/dev/V3.4__populate_city.sql

```diff
@@ -7,6 +7,7 @@ INSERT INTO City (id, country_id, region_id, district_id, name) VALUES ('35385',
 INSERT INTO City (id, country_id, region_id, district_id, name) VALUES ('35386','UA','2','39','Мощена');
 INSERT INTO City (id, country_id, region_id, district_id, name) VALUES ('35387','UA','2','39','Новий Мосир');
 INSERT INTO City (id, country_id, region_id, district_id, name) VALUES ('35388','UA','2','39','Нові Кошари');
+INSERT INTO City (id, country_id, region_id, district_id, name) VALUES ('42309','UA','10','185','Київ');
```

Comment:
The main rule is that you can't change the script that has already been applied. You should add new version instead.
It's ok for now, cuz this script hasn't been applied yet in prod.